src/util/audiosignal.h

```diff
@@ -51,6 +51,7 @@ class AudioSignal {
     static constexpr SINT kSamplingRate96kHz   = 96000;
     static constexpr SINT kSamplingRate192kHz  = 192000;
     static constexpr SINT kSamplingRateMax     = kSamplingRate192kHz; // upper bound
+    static constexpr SINT kSamplingRateMaxPlayback = kSamplingRate96kHz;
```

Comment:
I would like to see this constant at some central location within the engine code base instead of in this unbiased header file. If the engine decides to restrict the sample rate for playback it should define its own constant for this purpose.